base/dates/Dates.jl

```diff
@@ -1,5 +1,35 @@
 # This file is a part of Julia. License is MIT: http://julialang.org/license
 
+"""
+    Dates
+
+The `Dates` module provides `Date`, `DateTime`, `Time` types, and related functions.
+
+The types are not aware of time zones, based on UT seconds
+(86400 seconds a day, avoiding leap seconds), and
+use the proleptic Gregorian calendar, as specified in ISO 8601.
+For time zone functionality, see the TimeZones.jl package.
+
+```jldoctest
+julia> dt = DateTime(2017,12,31,23,59,59,999)
+2017-12-31T23:59:59.999
+
+julia> d1 = Date(Dates.Month(12), Dates.Year(2017))
+2017-12-01
+
+julia> d2 = Date("2017-12-31", Dates.DateFormat("y-m-d"))
+2017-12-31
+
+julia> Dates.yearmonthday(d2)
+(2017,12,31)
```

Comment:
BTW, running the doctest gave me an (apparently unrelated) error, so, err, I didn't.